### PR TITLE
(PC-23457) feat(search): use multi-search offers and venues when wipEnableVenuesInSearchResults feature flag activated

### PR DESCRIPTION
**Pull request by cedricls1**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-23457

## Checklist

I have:

- [ ] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion][1]

[1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

**Review by pierrecanthelou — COMMENTED** [review]
Merci ;)

Quelques retours

File: src/libs/algolia/fetchAlgolia/fetchOffersAndVenues/fetchOffersAndVenues.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      },
+    },
+    {
+      indexName: venuesIndex,
+      query: getSearchVenueQuery(parameters),
+      params: {
+        page: 0,
+        ...buildHitsPerPage(35),
+        ...(searchParameters.aroundLatLng ? { aroundLatLng: searchParameters.aroundLatLng } : {}),
+        aroundRadius: searchParameters.aroundRadius ?? 'all',
```
[the remark this answers is not in the document]
Comment (cedricls1) [reply src/libs/algolia/fetchAlgolia/fetchOffersAndVenues/fetchOffersAndVenues.ts]: `number | 'all'` (`SearchVenuePositionType`)

File: src/libs/algolia/fetchAlgolia/fetchOffersAndVenues/fetchOffersAndVenues.test.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import algoliasearch from 'algoliasearch'
+
+import { fetchOffersAndVenues } from 'libs/algolia/fetchAlgolia/fetchOffersAndVenues/fetchOffersAndVenues'
+import { SearchQueryParameters } from 'libs/algolia/types'
+import { env } from 'libs/environment'
+
+jest.mock('algoliasearch')
+
+const mockMultipleQueries = algoliasearch('', '').multipleQueries
```
[the remark this answers is not in the document]
Comment (cedricls1) [reply src/libs/algolia/fetchAlgolia/fetchOffersAndVenues/fetchOffersAndVenues.test.ts:9]: Je ne vois pas comment le faire :/ mais si tu as une idée je suis preneur

File: src/libs/algolia/fetchAlgolia/fetchOffersAndVenues/fetchOffersAndVenues.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (pierrecanthelou) [reply src/libs/algolia/fetchAlgolia/fetchOffersAndVenues/fetchOffersAndVenues.ts]: Ok

File: src/libs/algolia/fetchAlgolia/fetchOffersAndVenues/fetchOffersAndVenues.test.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (pierrecanthelou) [reply src/libs/algolia/fetchAlgolia/fetchOffersAndVenues/fetchOffersAndVenues.test.ts:9]: Pas trop non là comme ça :)

**Review by pierrecanthelou — APPROVED** [review]
Ok pour moi

**Review by YassinL — APPROVED** [review]

**Pull request merged**